.github/actions/detect-workflow-js/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "tsc",
     "format": "prettier --write '**/*.ts'",
     "format-check": "prettier --check '**/*.ts'",
-    "package": "ncc build --source-map",
+    "package": "ncc build --external encoding --source-map",
```

Comment:
Note to reviewers:

'encoding' is an optional package included as an optional "peerDependency" of `node-fetch` which is required by `octokit`. I'm not sure what makes `ncc` determine whether to include it in the dist/ or not but this option makes it so it's properly excluded.